### ISSUE TITLE
Changing the position of a component resets smalls of unchanged axis

### ISSUE THREAD

**Issue by subsoap** [defold/editor2-issues#2767]
![2019-09-30 06_19_30-spade png @ 100% (Layer 0, RGB_8)](https://user-images.githubusercontent.com/409170/65882740-76b26a80-e34a-11e9-8580-f7d9ea631ca7.png)

![2019-09-30 06_19_38-Faerie Solitaire Remastered - Defold Editor 2 0](https://user-images.githubusercontent.com/409170/65882748-7914c480-e34a-11e9-82bc-94cb11a0d218.png)

![2019-09-30 06_19_48-spade png @ 100% (Layer 0, RGB_8)](https://user-images.githubusercontent.com/409170/65882750-7b771e80-e34a-11e9-975a-af3f35b95d51.png)

Changing y position handle, resets z value to z.

**Comment by matgis-king** [comment]
Does it still do this if you turn off Move Whole Pixels from the Edit menu?

**Comment by subsoap** [comment]
@matgis-king 

Yes, in both cases this happens. In both cases I would expect it to no happen since I'm moving the Y axis widget and it is impacting the z value.
